rebar.config

```
@@ -6,6 +6,6 @@
 	{platform_define, "^R14", no_callbacks}
 ]}.
 {deps, [
-	{jsx, ".*", {git, "git@github.com:talentdeficit/jsx.git", {branch, master}}},
+	{jsx, ".*", {git, "git@github.com:talentdeficit/jsx.git", {branch, main}}},
```

I think we should switch to our own fork, right? This matches how core does it:

```suggestion
	{jsx, {git, "git@github.com:kivra/jsx.git", {tag, "2.9.0"}}},
```

---

yes, agreed!